### PR TITLE
PifPaf Wholebody documentation and visualization

### PR DESCRIPTION
**Pull request by DuncanZauss**
- Added the wholebody section to the guide
- Added a wholebody prediction image to the README and to the front page of the Guide

### CODE REVIEW

File: guide/plugins_wholebody.ipynb [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "6dfbbb5b",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "TypeError",
```
Comment (svenkreiss) [inline guide/plugins_wholebody.ipynb:92]: There is an error here.

File: openpifpaf/plugins/wholebody/wholebody.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
     pass
 
 
-class Wholebody(openpifpaf.datasets.DataModule):
+class Wholebody(openpifpaf.datasets.DataModule, openpifpaf.Configurable):
```
Comment (svenkreiss) [inline openpifpaf/plugins/wholebody/wholebody.py:29]: `Configurable` class is still experimental. Can you leave a brief comment why this is necessary here?

**Review by svenkreiss — COMMENTED** [review]
Thanks @DuncanZauss .

To fix cell tags (the thing that CI is complaining about):

```
cd guide
python nb_cell_tags.py
```

Please remove the png version of the soccer predictions. It's redundant with the jpeg one.

File: guide/plugins_wholebody.ipynb (hunk 1 above)
Comment (DuncanZauss) [reply guide/plugins_wholebody.ipynb]: Thank you! I incorporated all the comments in my last commits.

**Comment by svenkreiss** [comment]
Thanks!

**Pull request merged**